eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -67,4 +73,36 @@ protected function runQueriesFromFile($file)
             $this->db->exec($query);
         }
     }
+
+    /**
+     * Get DBMS-specific SQL data file path.
+     *
+     * @param string $relativeFilePath SQL file path relative to {baseDir}/{dbms} directory
+     *
+     * @return string absolute existing file path
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     *
+     * @since 6.13
+     */
+    final protected function getDatabasePlatformSpecificSQLFile($relativeFilePath)
+    {
+        $databasePlatform = $this->db->getDatabasePlatform()->getName();
+        $filePath = "{$this->baseDataDir}/{$databasePlatform}/{$relativeFilePath}";
+
+        $absoluteFilePath = realpath($filePath);
+        // check if realpath returned actual path, or false which means that file does not exist
+        if (false === $absoluteFilePath) {
```

Comment:
Simple `file_exists` would make strong type comparison (and comment) obsolete. You don't need absolute path anyway.

Reply:
Meant to prepare path for better console output, but indeed confusing. Fixed as agreed in bfee713.